src/data/nat/order/basic.lean

```diff
@@ -298,6 +277,14 @@ end
 | 0 := iff_of_false (lt_irrefl _) zero_le_one.not_lt
 | (n + 1) := lt_mul_iff_one_lt_left n.succ_pos
 
+lemma add_sub_one_le_mul {a b : ℕ} (ha : a ≠ 0) (hb : b ≠ 0) : a + b - 1 ≤ a * b :=
```

Comment:
I see a few lemmas that are not related to `dvd`. Are these supposed to be included? I can't quickly find any usages.